content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Configurability (Experimental)
+description: How to configure tracing options (experimental).
+weight: 60
+keywords: [telemetry,tracing]
+---
+
+Istio provides advanced capability to configure tracing options as of Istio 1.6.
+Previously, the only customization for tracing was the amount of randomly sampled requests.
+All of these are considered experimental.
+
+## Create a `MeshConfig` with trace settings
+
+All tracing options are configured by using `MeshConfig` during Istio installation.
+To simplify configuration, create a single YAML file to pass to `istioctl`.
+
+{{< text yaml >}}
+cat <<'EOF' > tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      tracing:
+EOF
+{{< /text >}}
+
+Then, you can append any configuration options to the `tracing.yaml` file.
+
+## Trace sampling
+
+Istio captures a trace for all requests by default when installing with the demo profile.
+For example, when using the Bookinfo sample application, every time you access
+`/productpage` you see a corresponding trace in the
+dashboard. This sampling rate is suitable for a test or low traffic
+mesh. For a high traffic mesh you can lower the trace sampling
+percentage in one of two ways:
+
+{{ warning }}
+Previously, the recommended method was to change the `values.pilot.traceSampling` setting during the mesh setup
+ or to change the `PILOT_TRACE_SAMPLE` environment variable in the pilot or istiod deployment.
+While this method to alter sampling continues to work, the following method
+is strongly recommended instead.
+
+In the event that both are specified, the value specified in the `MeshConfig` will override any other setting.
+{{ /warning }}
+
+To modify the default random sampling, which is defaulted to a value of 1, add the following to your
```

Comment:
A default of `1` doesn't match the text above for the `demo` profile. Do we want to clarify?